tools/setup_helpers/cmake.py

```diff
@@ -115,14 +115,14 @@ def _get_cmake_command():
         if IS_WINDOWS:
             return cmake_command
         cmake3 = which('cmake3')
-        if cmake3 is not None:
-            cmake = which('cmake')
-            if cmake is not None:
-                bare_version = CMake._get_version(cmake)
-                if (bare_version < LooseVersion("3.5.0") and
-                        CMake._get_version(cmake3) > bare_version):
-                    cmake_command = 'cmake3'
-        return cmake_command
+        cmake = which('cmake')
```

Comment:
This probably doesn't matter here, but this is a mild pessimization as now we have to do two which calls whereas previously we only did the second which call when cmake3 was not acceptable.